docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -197,7 +197,26 @@ implementation is used:
   - Set to `false` to disable in-memory caching of metadata files on the 
     coordinator. This cache is not used when `fs.cache.enabled` is set to true.
   - `true`
-
+* - `iceberg.expire_snapshots.min-retention`
+  -  Minimal retention period for `expire_snapshot` procedure. Equivalent
+     session property is `expire_snapshots_min_retention`.
+  -  `7 DAYS` 
+* - `iceberg.remove_orphan_files.min-retention`
+  -  Minimal retention period for `remove_orphan_files` procedure Equivalent
+     session property is `remove_orphan_files_min_retention`.
+  -  `7 DAYS`
+* - `iceberg.idle-writer-min-file-size`
+  -  Minimum data written by a single partition writer before it can
+     be considered as idle and can be closed by the engine. Equivalent
+     session property is `idle_writer_min_file_size`.
+  -  `16 MB`
+* - `iceberg.sorted-writing-enabled`
+  -  Enable sorted writing to tables with a specified sort order. Equivalent
+     session property is `sorted_writing_enabled`.
+  -  `true` 
+* - `iceberg.split-manager-threads`
+  -  Number of threads to use for generating splits.
```

Comment:
Whats the advice for users here?

Reply:
I don't follow the question.

We're talking about number of threads to do file scanning in Iceberg (fairly low-level detail): 
https://github.com/trinodb/trino/blob/77aca720b06f29475d002a82652a0a6fa839daa8/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplitManager.java#L136

Reply:
I guess the advice is the usual one .. dont touch.